docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -401,6 +415,489 @@ docker compose up
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+:::caution
+
+It's possible to send multiple transactions to fulfill user request, but due to current protocol restrictions, only the latest one will be visible for user via `stellar_transaction_id` field.
+
+:::
+
+#### Handling Errors
+
+When unrecoverable error happens, it's required to set transaction status to `error`. Message field can be used to describe error details.
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "id": "<transaction_d>",
+  "status": "error",
+  "message": "Example error"
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+:::tip
+
+This status should not be used after user has made a transfer (if there are some external factors that prevent completing transaction). Instead, refund should be initiated. It can be done either by user's request (in your web view), or automatically.
+
+:::
+
+### Updating Withdrawal Transaction Via `PATCH`
+
+Once the deposit flow is finished, implementing withdrawal becomes quite easy. Some parts of the flow are similar, and can be reused.
+
+The starting point for withdrawal and deposit is the same, i.e., [collecting and verifying KYC](#kyc-being-collectedverified).
+
+#### Ready to Receive Funds
+
+Similar to deposit, the next step would be to notify the user that the Anchor is ready to receive funds. However, as your service will be receiving transactions over the Stellar network, the update would look a bit different
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "transaction": {
+    "id": '"$transaction_id"',
+    "status": "pending_user_transfer_start",
+    "message": "waiting on the user to transfer funds",
+    "kyc_verified": "true",
+    "amount_in": {
+      "asset": "stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5",
+      "amount": "10"
+    },
+    "amount_out": {
+      "asset": "iso4217:USD",
+      "amount": "9.0"
+    },
+    "amount_fee": {
+      "asset": "stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5",
+      "amount": "1.0"
+    },
+    "memo": "466857256912784999",
+    "memo_type": "id"
+  }
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+:::tip
+
+Setting `memo` and `memo_type` is optional. When status is changed to `pending_user_transfer_start`, Anchor Platform sets `memo` and `memo_type` automatically (only if it's not included in the request). Note that the memo must be unique, this is what helps to associate stellar transactions with SEP transaction.
+
+:::
+
+:::note
+
+The Stellar account that will be used to receive funds should be configured in [assets](#specify-your-services-assets)
+
+:::
+
+#### Tracking Stellar Transaction
```

Comment:
Instead of suggesting it in a tip, I would definitely guide businesses to use the observer unless they already have a system for monitoring payments made to their on-chain accounts.